core/tee/tee_svc.c

```diff
@@ -70,7 +70,11 @@ static const char descr[] = TO_STR(CFG_TEE_IMPL_DESCR);
  * The real-time clock MUST be out of reach of software attacks
  * from the REE.
  */
+#ifdef CFG_SECURE_TIME_SOURCE_REE
 static const uint32_t ta_time_prot_lvl = 100;
+#elif CFG_SECURE_TIME_SOURCE_CNTPCT
```

Comment:
`CFG_SECURE_TIME_SOURCE_CNTPCT=y` is not sufficient here as the time reference is lost across reboots.

Reply:
thanks,Currently OpTEE-OS is not implemented to meet the GP 1000 level,so let it alway 100 level?